### DISCUSSION TITLE
Flexget\Dockerfile

### DISCUSSION THREAD

**Discussion by paulpoco** [Flexget/Flexget#2829] [Q&A]
I have my own Docker for Unraid DelugeVPN with Flexget. 

I was looking at the Flexget\Dockerfile and was looking at pulling out my install of Flexget from the install files of DelugeVPN and do it all in the Dockerfile,  is that possible?   It is based on Binhex-DelugeVPN's Docker, I notice his now has to option in the docker-template to manually add your ports to the install(flexget web-ui), currently I do inside the different files.  Now Binhex uses his own base like 
"FROM binhex/arch-int-vpn:latest".

My github link is
https://github.com/paulpoco/arch-delugevpn

@X-dark do you have any input?   Since I see you are the Author of the Flexget\Dockerfile.

**Comment by X-dark** [comment]
I am not sure to understand what you are trying to do. If you want to run several process in the same container, forget it, this is a really bad idea.

**Reply by paulpoco** [reply]
I currently run Deluge, OpenVPN, Flexget in same container. 

I was trying to find a cleaner way to do it.

**Comment by ksurl** [comment]
I would use one of the existing popular images for flexget such as wiserain or cpoppema and point the config to your deluge vpn container ip. Just make sure the vpn container has allowed connections from your local network. It is cleaner to manage applications in separate containers. The only reason there are some combined with vpn is for the guaranteed killswitch though you could achieve the same thing with a network linked container
